setup.cfg

```diff
@@ -34,6 +34,11 @@ universal = 1
 [flake8]
 max-line-length = 99
 exclude = docs, .tox, .git, __pycache__, .ipynb_checkpoints
+inline-quotes = single
+
+[pydocstyle]
+convention = google
+add-ignore = D107, D407, D417
```

Comment:
On a previous PR you used a different format that I liked a lot: multiple lines for the ignores, with a comment alongside each one mentioning what it was. I think that is very practical and makes it easier to maintain in the future.
Maybe we could do it here too?

Reply:
Sure. I'm actually moving `pydocstyle` to its own PR though, because it doesn't pass in the existing code.